frontend/styles/components/calendar/mobile/DayLandscapeView.module.scss

```
@@ -0,0 +1,179 @@
+@import '../../../Variables.module.scss';
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)